content/en/docs/Writing policies/generate.md

```diff
@@ -26,6 +26,18 @@ Kubernetes has many default resource types even before considering CustomResourc
 When generating a custom resource, it is necessary to set the apiVersion (ex., `spec.generate.apiVersion` and kind (ex., `spec.generate.kind`).
 {{% /alert %}}
 
+To get the details and status of generated resource check the details of generate request. The following command will give the list of generate requests.
+
+```sh
+kubectl get generaterequests.kyverno.io -A
+```
+
+Generate request status can have four values:
+`Completed` - the generate request controller created resources defined in the policy
+`Failed` - the generate request controller failed to process the rules
+`Pending` - the request is yet to be processed or resource has not been created
+`Skip` - the generate request controller has skipped the creation of resource
```

Comment:
Is the `Skip` only be used when the source resource already exists in the cluster while the policy does not have a label/annotation selector defined?

Reply:
for now yes. But skip can be used as a generic status. And when we do so this can be updated.

Reply:
I don't see other use cases that we need to mark as `Skip`. If the source does not match, why bother create a GR and mark as `Skip`?

Reply:
please check [this](https://github.com/kyverno/kyverno/pull/2657) PR 
the GR is created before and then processed for resource creation. This is skipped [here](https://github.com/kyverno/kyverno/blob/5195bc5bf298a9d81252c349e4231a7b91cbaec7/pkg/generate/generate.go#L297), without any error.

Reply:
Isn't this the case I described above?
>Is the Skip only be used when the source resource already exists in the cluster while the policy does not have a label/annotation selector defined?

Reply:
Yes. And when the preexisting resource is updated the new resource is not generated.

Reply:
Is this resolved?

Reply:
Is this resolved?